sherpa/astro/io/crates_backend.py

```diff
@@ -525,18 +547,18 @@ def get_column_data(*args):
 
     cols = []
     for arg in args:
-        if isinstance(arg, CrateData):
+        if isinstance(arg, pycrates.CrateData):
             # vals = arg.get_values()
             vals = arg.values
 
-        elif arg is None or isinstance(arg, (numpy.ndarray, list, tuple)):
+        elif arg is None or isinstance(arg, (np.ndarray, list, tuple)):
```

Comment:
This almost, but not exactly looks like

```suggestion
        elif arg is None or np.isiterable(arg):
```
I suspect that is what this should be to be general.

Reply:
I agree that it looks sensible, it's just I don't want to make this change here "just in case". I admit that it is irrational (I have made bigger changes elsewhere in this module)!